alws/crud/build.py

```diff
@@ -85,7 +85,7 @@ async def create_build(
         for flavour in flavors:
             db_build.platform_flavors.append(flavour)
     db.add(db_build)
-    await db.commit()
+    await db.flush()
     await db.refresh(db_build)
```

Comment:
As far as i know, exception is thrown in case of calling `refresh` on a deleted object, it's not the case here. Also this method is tested several times in `tests/test_builds`, it works fine